packages/demo/package.json

```diff
@@ -54,6 +54,7 @@
     "@web3-onboard/web3auth": "^2.2.2",
     "@web3-onboard/xdefi": "^2.0.3",
     "@web3-onboard/zeal": "^2.0.3",
+    "react": "^18.2.0",
```

Comment:
Found that the demo doesn't build unless you've got react globally installed. Have added it explicitly as a dependency

Reply:
Please also up "@web3-onboard/injected-wallets": "^2.9.1-alpha.1", 
in the demo for testing